src/SDK/Trace/Behavior/LoggerAwareTrait.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Trace\Behavior;
+
+use Psr\Log\LoggerAwareTrait as PsrTrait;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+use Psr\Log\NullLogger;
+
+trait LoggerAwareTrait
+{
+    use PsrTrait;
+
+    private string $defaultLogLevel = LogLevel::INFO;
```

Comment:
good time to implement `OTEL_LOG_LEVEL` per spec? (default is `info` if not set)

Reply:
Hmm, the default is 'info' at the moment, so that's according to spec (by accident). 
However if you mean, the trait/class should pull the value of the env variable `OTEL_LOG_LEVEL` out of the global scope, that's not something I want to implement. I am personally a strong believer in the Single Responsibilty Principle. So a class should be stupid and concentrate on it's job. It should not even know env vars exist, unless its job is to deal with env vars. I would prefer a solution where such config variables are resolved by a dedicated component and are then populated to the places which need them.

But again, these are just my thoughts.

Reply:
That makes sense - I'm thinking ahead to when I am instrumenting code (we have literally a couple hundred small to mid-sized applications), where I want the library to configure itself as much as possible with environment variables. But I agree that it's not the responsibility of this logger to care about that.

Reply:
Ok, got you. I thought about something like that as well. But a lot of frameworks have their own way how to deal with env vars, which have to be taken into account as well. It's probably a bit out of the scope of this PR, we can deal with this later.

Reply:
@bobstrecansky
Are there any plans (or active development) regarding configuration of the SDK? The configuration of the SdkBundle could be extracted out for this purpose (with some changes) and this could also take care of the whole env var issue.